### ISSUE TITLE
 Black Activities Overview After Wake Up 

### ISSUE THREAD

**Issue by EmreKaratopuk** [micheleg/dash-to-dock#2325]
I am using Ubuntu 24.04.1 and it's a fresh install. The problem is when I suspend and wake up my computer, the activities overview doesn't work and it's just black screen. The alt-tab screen is also black. I tried to disable the dash to dock extension and disabling solved the problem. Therefore the problem seems dash to dock specific. Please see the images below for more info. 

![Image 1](https://github.com/user-attachments/assets/fc68e19b-302a-4487-a82a-d44f39651b09)

![Image 2](https://github.com/user-attachments/assets/7630eb13-0dcc-441c-bed8-54dddc48bd28)

![Image 3](https://github.com/user-attachments/assets/577fd160-a527-4a7a-aa1e-55b9169d6aaf)

**Comment by sergio-costas** [comment]
What happens if you disable all the extensions and reboot? Does it "fixes" the problem? If that's the case, can you enable the extensions one by one to see which one produces that problem?

**Comment by EmreKaratopuk** [comment]
I have already tried that and the problem occurred again when I enabled dash-to-dock. However, I've been using gnome without enabling dash-to-dock for several days already, and the issue is gone. Therefore I think the problem is about dash-to-dock extension.

**Comment by sergio-costas** [comment]
Yes, it makes sense. Sorry. I edited my previous comment. I'll try to reproduce the bug.

**Comment by sergio-costas** [comment]
BTW: does it happen when you suspend from the desktop, or only if you suspend from the Activities view?

**Comment by sergio-costas** [comment]
I've been unable to reproduce it. Can you open a terminal, type

    sudo journal -f

suspend and wake up the computer, and see if there are any message about gnome-shell or dash-to-dock with errors?

**Comment by vanvugt** [comment]
Since you're using Ubuntu, the problem is likely #1992. If you want to use dash-to-dock then please also disable ubuntu-dock.

**Comment by vanvugt** [comment]
Closing due to no response from the original reporter.
